### PR TITLE
Adding articles to seed.rb

### PR DESCRIPTION
**Pull request by jennymalmefeldt**
# This PR contains:

## [PT](https://www.pivotaltracker.com/story/show/175232851)

- Fake data to seed.rb

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+articles = Article.create(
+  [
+    {
+
```
Comment (johanperjulius1) [inline db/seeds.rb]: Format/ remove excessive line break

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+    },
+    {
+      title: "The Delicacies Created Because of Coronavirus",
+      category: "entertainment",
+      teaser: "As people continue to avoid leaving home as much as possible, new creative levels are being reached in the kitchen.",
+      content: "We are in a time of Rennasaince for the home chef as people continue to be reticient to leave their home unless absolutely necessary. People are finally finding uses for the mostly empty bags of frozen vegetables, that last half cup of flour, and only having enough pasta for half a meal. They are taking the normally ignored food items that never seem to make their way to a plate, and creating wonderfully diverse, if sometimes inedible, creations that are delighting the world. Stay tuned for an exclusive of some of these wonderfully unexpected and bizarre meals."
+
```
Comment (johanperjulius1) [inline db/seeds.rb]: Remove excessive line break

**Review by johanperjulius1 — CHANGES_REQUESTED** [review]

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,34 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+articles = Article.create(
```
Comment (emtalen) [inline db/seeds.rb]: Why are you wrapping it in a variable? Are you using it for anything?

File: db/seeds.rb [hunk 4]
```diff
@@ -1,7 +1,34 @@
...
+      teaser: 'We are all in quarentine',
+      content: 'We are all mad here. That quote from the Mad Hatter in Alice in Wonderland has never been more true than now. We are experiencing a wave of maddness as the quarentine continues for people across the globe. People in Italy have been reported to actually talk to their neighbors, playing music for the street and even group calisthenics on the patio. These are indeed scary times. As we move forward into an uncertain future, we have got to wonder, what will the Italians do next.'
+    },
+    {
+      title: 'Thomas Got a New Car',
+      category: 'sports',
+      teaser: 'He bought it to comfort himself',
+      content: 'On Wednesday of last we, we here at Newsroom 1 were informed that Thomas Ochman had bought a new car. And not just any car but a Berlingo. His son, Oliver, was reported to have shaken his head in sadness when he heard the news, questioning why his father needed another car. And yet, later that same day we also learned that Mr. Ochman wants a third car. And not just any car, but a blank that he found on Blocket. The big questions now are: will he get it and how will Oliver react?'
+    },
```
Comment (emtalen) [inline db/seeds.rb:14]: shame on you for stealing content 😉

File: db/seeds.rb (hunk 3 above)
Comment (jennymalmefeldt) [reply db/seeds.rb]: I thought it looked cleaner, to not repeat the code in every block.

File: db/seeds.rb (hunk 4 above)
Comment (jennymalmefeldt) [reply db/seeds.rb:14]: Thanks for sharing 😉

File: db/seeds.rb (hunk 3 above)
Comment (emtalen) [reply db/seeds.rb]: yes, I agree but do you need to wrap in `articles =` ?

**Review by emtalen — APPROVED** [review]

**Pull request merged**